rust/src/lib.rs

```diff
@@ -95,7 +95,7 @@ impl UnitInterval {
 type SubCoin = UnitInterval;
 type Rational = UnitInterval;
 type Epoch = u32;
-type Slot = u32;
+type Slot = u64;
```

Comment:
For use from wasm it would be better if we could use `BigNum` instead of `u64`, as `u64` only bindgens to JS `BigInt` in some environments, but doesn't work from others as js numbers only support ints up to I think 53 bits (IEEE doubles).